components/api_version/src/api_v2.rs

```diff
@@ -198,11 +197,11 @@ impl KvFormat for ApiV2 {
     ) -> (Vec<u8>, Vec<u8>) {
         match src_api {
             ApiVersion::V1 | ApiVersion::V1ttl => {
-                start_key.insert(0, RAW_KEY_PREFIX);
+                start_key = ApiV2::add_prefix(&start_key);
                 if end_key.is_empty() {
                     end_key.insert(0, RAW_KEY_PREFIX_END);
```

Comment:
The EMPTY end key should be mapped to `r001` or `x001`?.

Reply:
Another question, when do we need this function while executing backup and restore.

Reply:
This is a conversion scenario, no key space exists in apiv1, so I think we need set 'r' + 1 as the whole end. Anyway, 'r' + 1 and `r001` both works here. @pingyu How do you think?

Reply:
This function is needed when return backup response, range in `BackupResponse.File` need to be the apiv2 format.

Reply:
En... I suggest `r001` . It is a reasonble range start & end for default key space. Although both are ok here.

Reply:
After consideration, i think use `r001` as the end is better. `r001` as end can avoid some useless download and ingest during restore if target tikv cluster is not empty. I have changed the end as `r001`. PLAT~ @iosmanthus @pingyu